### PR TITLE
fix(lending): skip token in getPortfolios if not found in getReserveTokens

### PR DESCRIPTION
**Pull request by LuPoYi**
- [x] changeset
- [x] skip token in getPortfolios if not found in getReserveTokens

### CODE REVIEW

File: packages/lending/src/protocols/aave-v2/lending-protocol.ts [hunk 1]
```diff
@@ -345,51 +345,56 @@ export class LendingProtocol extends Protocol {
...
-      const assetPrice = assetPriceMap[token.address];
-      const userBalance = userBalancesMap[token.address];
+      if (!reserveData) continue;
 
-      let usageAsCollateralEnabled = reserveData.usageAsCollateralEnabled;
-      if (Number(userBalance.supplyBalance) > 0) {
-        usageAsCollateralEnabled = usageAsCollateralEnabled && userBalance.usageAsCollateralEnabled;
-      }
+      const {
+        usageAsCollateralEnabled: _usageAsCollateralEnabled,
```
Comment (Jeff-CCH) [inline packages/lending/src/protocols/aave-v2/lending-protocol.ts]: follow the naming convention of Ln360, should _usageAsCollateralEnabled be `reserveDataUsageAsCollateralEnabled`?

Comment (LuPoYi) [reply packages/lending/src/protocols/aave-v2/lending-protocol.ts]: used `_usageAsCollateralEnabled` to keep it shorter,  `reserveDataUsageAsCollateralEnabled` works for me too, lets keep the same naming convention.

Comment (LuPoYi) [reply packages/lending/src/protocols/aave-v2/lending-protocol.ts]: updated

**Review by Jeff-CCH — APPROVED** [review]
LGTM

**Pull request merged**